### PR TITLE
user name changes on login as necessary

### PR DESCRIPTION
**Pull request by RafaelELopez**

### CODE REVIEW

File: app/controllers/login.js [hunk 1]
```diff
@@ -30,11 +30,21 @@ function login(req, res, next) {
 
 			// Return that user if they exist
 			if (user) {
-				auth.signToken({user_id: user.attributes.id}, (err, userJwt) => {
-					return response.ok(res, {
-						jwt: userJwt
-					});
-				});
+				let userPromise;
+				// Update the user's name if it changed
```
Comment (JDonut) [inline app/controllers/login.js]: Move this comment above the `userPromise` declaration.
Also it should mention that the name changed on Orcid, like "Update the user's name if they changed it in Orcid's system`

File: app/controllers/login.js [hunk 2]
```diff
@@ -30,11 +30,21 @@ function login(req, res, next) {
...
+					userPromise = user.set({name: userName}).save()
+				}
+				else {
+					userPromise = Promise.resolve(user);
+				}
+				return userPromise.then(resolvedUser => {
+					auth.signToken({user_id: resolvedUser.attributes.id}, (err, userJwt) => {
+						return response.ok(res, {
+							jwt: userJwt
+						})
```
Comment (JDonut) [inline app/controllers/login.js]: Missing semicolon

File: app/controllers/login.js [hunk 3]
```diff
@@ -30,11 +30,21 @@ function login(req, res, next) {
...
+					})
```
Comment (JDonut) [inline app/controllers/login.js]: ;

File: app/controllers/login.js [hunk 4]
```diff
@@ -30,11 +30,21 @@ function login(req, res, next) {
...
+				})
```
Comment (JDonut) [inline app/controllers/login.js]: ;

File: app/controllers/login.js [hunk 5]
```diff
@@ -30,11 +30,21 @@ function login(req, res, next) {
...
-					return response.ok(res, {
-						jwt: userJwt
-					});
-				});
+				let userPromise;
+				// Update the user's name if it changed
+				if(user.get('name') !== userName){
+					userPromise = user.set({name: userName}).save()
+				}
+				else {
```
Comment (JDonut) [inline app/controllers/login.js]: Pull this up a line, like `} else {...`

**Comment by RafaelELopez** [comment]
Adressed your suggestions/requests. Thanks for noticing those.

**Pull request merged**